openforcefield/utils/toolkits.py

```diff
@@ -142,8 +157,6 @@ class ToolkitWrapper:
     _is_available = None  # True if toolkit is available
     _toolkit_name = None  # Name of the toolkit
     _toolkit_installation_instructions = None  # Installation instructions for the toolkit
-    _toolkit_file_read_formats = None  # The file types that this toolkit can read
-    _toolkit_file_write_formats = None  # The file types that this toolkit can write
```

Comment:
There was ambiguity as to whether these were CLASS or INSTANCE methods. RDKit did something silly where this is actually a dict like `{'sdf':rdkit.Chem.SDFWriterClass}`, so even so much as putting the class definition through the interpreter would raise an exception if rdkit wasn't available. Since there are a few cases where this could benefit from being modified, I figure it's best to switch it over to an instance attribute (defined in `__init__`)